### PR TITLE
web-console: add note about CSV import throught COPY command

### PR DESCRIPTION
**Pull request by argshook**
This PR updates web console import dialog with a note about `COPY`
command

image

### CODE REVIEW

File: packages/web-console/src/partials/import.hbs [hunk 1]
```diff
@@ -28,61 +28,51 @@
...
-                style="margin: 0 15px 20px 0;"
+                style="margin: 0 15px 20px 0"
               ></i>
+              <i class="fa fa-file-alt fa-3x" style="margin: 0 15px 20px 0"></i>
             </div>
-            <h2 style="font-weight: 500;">
-              Drag files or paste data here to
-              import
+            <h2 style="font-weight: 500">
+              Import CSV file
```
Comment (amyshwang) [inline packages/web-console/src/partials/import.hbs:39]: Seeing the screenshot, I wonder if this line is needed at all. 
Should we consider changing this to "click here to browse" with the link?

**Review by amyshwang — COMMENTED** [review]
Some more thoughts but overall looks good.

File: packages/web-console/src/partials/import.hbs (hunk 1 above)
Comment (argshook) [reply packages/web-console/src/partials/import.hbs:39]: I actually tried exactly that, but it felt a bit empty, I feel like some explanation of a CTA makes sense.

Wdyt?

Comment (argshook) [reply packages/web-console/src/partials/import.hbs:39]: image

Comment (amyshwang) [reply packages/web-console/src/partials/import.hbs:39]: Actually you're right! I think the original one looks better!

**Review by amyshwang — APPROVED** [review]
Based on the screenshot it looks good.

**Pull request merged**